remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,32 @@
+import remotespark.utils.configuration as conf
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        events_handler = self._import_handler(conf.events_config()["handler"])
+        self.handler = events_handler()
+
+    def _import_handler(self, name):
+        """
+        Importing the handler based on the passed parameter.
+        """
+        components = name.split('.')
+        mod = __import__(components[0])
```

Comment:
Yes, but it's still an internal implementation detail that shouldn't be used by us:

> Import a module. Because this function is meant for use by the Python interpreter and not for general use it is better to use importlib.import_module() to programmatically import a module.